backend/views.py

```diff
@@ -21,3 +21,13 @@ def addTask(request):
     except Exception as e:
         return JsonResponse({'fail': 'Task creation failed'})
     return JsonResponse({'success': True})
+
+@api_view(['PUT'])
+def updateTask(request):
+    try:
+        updatedTask = Task.objects.get(id=request.data['id'])
```

Comment:
more like taskToUpdate, it's not updated yet :)

Reply:
@Kotauror's comment was about the variable that you have defined on line 28, not the route handler name on line 26.

```py
taskToUpdate = Task.objects.get(id=request.data['id'])
```